forge-gui/res/cardsfolder/upcoming/ashnod_the_uncaring.txt

```diff
@@ -0,0 +1,10 @@
+Name:Ashnod the Uncaring
+ManaCost:2 U B R
+Types:Legendary Creature Human Artificer
+PT:1/4
+K:Deathtouch
+T:Mode$ AbilityCast | ValidCard$ Artifact.inZoneBattlefield,Creature.inZoneBattlefield | ValidSA$ SpellAbility.nonManaAbility | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Condition$ Sacrificed | Execute$ TrigCopy | OptionalDecider$ You | TriggerDescription$ Whenever you activate an ability of an artifact or creature that isn't a mana ability, if one or more permanents were sacrificed to activate it, you may copy that ability. You may choose new targets for the copy. (Sacrificing an artifact for mana to activate an ability doesn't count.)
```

Comment:
does it needs to be `Activated.nonManaAbility` or does AbilityCast already only works for Activated?

Reply:
Hmm, I'm not sure – I stole that line straight from Crackdown Construct which looks like it was tweaked by @tool4ever in 573e7df

Reply:
Only activated:
https://github.com/Card-Forge/forge/blob/f67fe035f5eda04a21d23dd35c7574063bcffc46/forge-game/src/main/java/forge/game/zone/MagicStack.java#L256